src/PlayFabBuddy.Lib/Interfaces/Adapter/IPlayStreamAdapter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using PlayFabBuddy.Lib.Adapter.Accounts;
+
+namespace PlayFabBuddy.Lib.Interfaces.Adapter;
+
+public interface IPlayStreamAdapter
+{
+    /// <summary>
+    /// All Players
+    /// </summary>
+    public const string AllPlayersSegmentName = "All Players";
```

Comment:
Can't find any concrete usages for these "strings" in the project except this one used as documentation for the command settings. 

Are these necessary at all? If the intention is to document here maybe, we can find a better place or style for it? Enum, Entity or Dictonary?

Line 15, 20, 25 are "included" in this comment :D

Reply:
see https://github.com/Annonator/PlayFabBuddy/pull/52#discussion_r801009309

You opting to remove the doc comments altogether? Or just want me to make them "better"?

Reply:
I don't think that we should have these constants here just to reference them in documentation if we are not validating them in code. 

As the segment names might be dynamic and user managed any documentation here in code might be outdated for a given users project. I'm not sure if we can make them bette in this case and I would delete them.